Can't find system headers

/usr/include/sys/types.h:146:10: fatal error: 'stddef.h' file not found
    /usr/include/sys/types.h:146:10: fatal error: 'stddef.h' file not found, err: true

Looks like you go to the trouble of figuring out where to look - https://github.com/servo/rust-bindgen/blob/bdfab643e70df469dd0d163cc91bf48955250735/src/bin/bindgen.rs#L64-L67 - and then forget all about it! - https://github.com/servo/rust-bindgen/blob/bdfab643e70df469dd0d163cc91bf48955250735/src/bin/bindgen.rs#L73.

I think the fix is simply:

    -    match builder_from_flags(env::args()) {
    +    match builder_from_flags(bind_args.into_iter()) {

--- a/src/bin/bindgen.rs
+++ b/src/bin/bindgen.rs
@@ -70,7 +70,7 @@ pub fn main() {
         }
     }
 
-    match builder_from_flags(env::args()) {
+    match builder_from_flags(bind_args.into_iter()) {
         Ok((builder, output)) => {
             let mut bindings = builder.generate()
                 .expect("Unable to generate bindings");

